mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job.go.erb

```diff
@@ -470,12 +553,12 @@ func resourceDataflowFlexTemplateJobUpdate(d *schema.ResourceData, meta interfac
 	request := dataflow.LaunchFlexTemplateRequest{
 		LaunchParameter: &dataflow.LaunchFlexTemplateParameter{
 
-			ContainerSpecGcsPath: d.Get("container_spec_gcs_path").(string),
-			JobName:              d.Get("name").(string),
-			Parameters:           tpgresource.ExpandStringMap(d, "parameters"),
+			ContainerSpecGcsPath:  d.Get("container_spec_gcs_path").(string),
+			JobName:               d.Get("name").(string),
+			Parameters:            tpgresource.ExpandStringMap(d, "parameters"),
```

Comment:
Why don't we need `updatedParameters` on an update?

Reply:
Ah, my bad! I had updated to using the `updatedParameters` as one of the final changes to resolve a creation case but the update case I did afterward did not catch when this would cause problems